users/models.py

```diff
@@ -1,12 +1,22 @@
 # -*- coding: utf-8 -*-
-from __future__ import unicode_literals
-from django.urls import reverse
 from django.db import models
+from django.contrib.auth.models import User
+from django.db.models.signals import post_save
+from django.dispatch import receiver
 
 
-class User(models.Model):
-    first_name = models.CharField('First name', max_length=30, blank=False)
-    last_name = models.CharField('Last name', max_length=30, blank=False)
-    email = models.EmailField(max_length=70, unique=True)
-    nick_name = models.CharField('Nick name', max_length=30, blank=False)
-    user = models.CharField('Username', max_length=30, blank=False)
+class Profile(User):
+    user = models.OneToOneField(User, on_delete=models.CASCADE)
+    nick_name = models.CharField(
+        'Nick name', max_length=30, blank=True, null=True)
+
+
+@receiver(post_save, sender=User)
+def create_user_profile(sender, instance, created, **kwargs):
+    if created:
+        Profile.objects.create(user=instance)
+
+
+@receiver(post_save, sender=User)
+def save_user_profile(sender, instance, **kwargs):
+    instance.profile.save()
```

Comment:
I think the @receiver code is what's breaking your super user. We shouldn't really be using signals and receivers for this app. It's not really that complex to warrant that use. Why do we need them here?